.github/workflows/tests.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9, '3.10', '3.11']
```

Comment:
The update to a recent version of `nbconvert` requires Python >= 3.8. I also checked and 3.7 was EOL back in June (see: https://devguide.python.org/versions/). It would have been great to not drop 3.7 in the testing, but this seems to be the answer to the rabbit hole of dependencies. (The root of the issue comes from the `tests.yml` workflow installing the core, test and docs dependencies, and therefore needing all three to be consistent with one another)